kyo-core/jvm/src/main/scala/kyo/KyoAppPlatformSpecific.scala

```diff
@@ -0,0 +1,15 @@
+package kyo
+
+import scala.collection.mutable.ListBuffer
+
+abstract class KyoAppPlatformSpecific extends KyoApp.Base[Async & Resource & Abort[Throwable]]:
```

Comment:
The build is failing in Native. I think it's just a matter of moving this file to `kyo-core/jvm-native/src/`

Reply:
updated and tested, should work now